src/main/java/com/shopping_app/shoppingApp/model/Cart/CartItemResponse.java

```diff
@@ -0,0 +1,29 @@
+package com.shopping_app.shoppingApp.model.Cart;
+
+import com.shopping_app.shoppingApp.domain.CartItems;
+import com.shopping_app.shoppingApp.model.Product.ProductResponse;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Getter
+@Setter
```

Comment:
Do we need this annotations? If the code works without these annotations, you can remove it. It's good to add only code that is used

Reply:
will check and remove the unnecessary annotations